includes/wc-update-functions.php

```diff
@@ -2242,3 +2242,41 @@ function wc_update_450_sanitize_coupons_code() {
 	delete_option( 'woocommerce_update_450_last_coupon_id' );
 	return false;
 }
+
+/**
+ * Add empty meta attribute values for variable products having a value of "Any..." for the attribute,
+ * only for the variations missing them, in increments of 100 products.
+ *
+ * @return bool true if there are more products to process.
+ */
+function wc_update_450_insert_attribute_metadata_for_variations_not_having_it() {
```

Comment:
any way to work around this migration? this is going to really really long time for many hosts, let's go an extra mile to avoid this if we can.

Reply:
But we are using those empty meta values to detect "Any" variations. If we don't ensure that these meta values exist for all variations, the widget won't work properly.

Reply:
Resolving because we discussed offline and figured out a way to not need this migration.